### PR TITLE
Fixes

### PR DESCRIPTION
**Pull request by suhimanju**
## List of changes

1. Added scripts for binding secret texts and files
2. Added scripts for jenkins jobs backup
3. Minor Selenium fixes

### CODE REVIEW

File: Class3/selenium_iframes.py [hunk 1]
```diff
@@ -9,14 +9,14 @@
 # Create a webdriver instance for chrome browser
 browser = webdriver.Chrome()
 
-browser.get('file:///home/arch-suhi/techsparks/python_ci_automation/Class3/iframe.html')
```
Comment (machzqcq) [inline Class3/selenium_iframes.py:12]: No hardcoding file paths like this. Please have the html file checked into source and refer to that as relative path

File: Class3/selenium_website_links.py [hunk 2]
```diff
@@ -0,0 +1,41 @@
+import time
+from selenium import webdriver
+
+# Function that prints the html tag of all the links (<a> tags) on the website. Default webpage is: https://www.inmar.com
+def extract_links(url="https://www.inmar.com"):
```
Comment (machzqcq) [inline Class3/selenium_website_links.py]: Don't use inmar website please. There are many more out there

**Comment by suhimanju** [comment]
Will correct them and check-in the changes. Thanks for pointing out.

File: Class3/selenium_iframes.py (hunk 1 above)
Comment (suhimanju) [reply Class3/selenium_iframes.py:12]: Made required changes by refering to relative path

File: Class3/selenium_website_links.py (hunk 2 above)
Comment (suhimanju) [reply Class3/selenium_website_links.py]: My bad. Just focused on script pass/fail. Hence, went unnoticed. Removed inmar in all the places.

**Review by machzqcq — APPROVED** [review]

**Pull request merged**